pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/model/defaultvalues/ReferralSourceServiceTest.java

```diff
@@ -285,7 +284,7 @@ private void applyFoundOutAnswer(List<Activities> activities, List<String> answe
         refSourceQA.put(DDPActivityConstants.ACTIVITY_QUESTION_ANSWER, answer);
     }
 
-    protected static List<Activities> getActivities() {
+    public static List<Activities> getActivities() {
```

Comment:
Candidate for `@VisibleForTest`?

Reply:
This is already a test, so unnecessary.
At some point I will reorganize this and similar into test utilities.

Reply:
doh, sorry I totally lost that this was already in a test!